### PR TITLE
Checkpoint 5

### PR DESCRIPTION
**Pull request by DarylMcCullough**
I'm not sure if the to-do items are supposed to have a body, or description, (or both) in addition to their names. If so, I should make an update.

### CODE REVIEW

File: app/controllers/items_controller.rb
Comment (brittanyjspringer) [inline app/controllers/items_controller.rb]: I think you meant Item here.

File: app/helpers/items_helper.rb
Comment (brittanyjspringer) [inline app/helpers/items_helper.rb:1]: you will end up using this so I'll allow it.

File: app/views/items/new.html.erb
Comment (brittanyjspringer) [inline app/views/items/new.html.erb]: This can be removed?

File: app/views/items/create.html.erb
Comment (brittanyjspringer) [inline app/views/items/create.html.erb]: Made me laugh but definitely needs to be removed.

File: config/routes.rb
Comment (brittanyjspringer) [inline config/routes.rb]: I would typically next my Items under my User like this: 

```
  resources :users, only: [:show] do
    resources :items, only: [:create, :destroy]
  end
```

your call if that makes sense with your workflow.

File: spec/models/item_spec.rb
Comment (brittanyjspringer) [inline spec/models/item_spec.rb:1]: This should be filled in with some tests.

File: spec/helpers/items_helper_spec.rb
Comment (brittanyjspringer) [inline spec/helpers/items_helper_spec.rb]: Remove please.

File: spec/views/items/new.html.erb_spec.rb
Comment (brittanyjspringer) [inline spec/views/items/new.html.erb_spec.rb]: Remove please.

**Review by brittanyjspringer — CHANGES_REQUESTED** [review]

**Comment by brittanyjspringer** [comment]
Review submitted with changes requested. Back to you, @DarylMcCullough.

File: config/routes.rb
Comment (DarylMcCullough) [reply config/routes.rb]: I tried this, and I think it would take a lot of work to make it do the right thing. The most immediate problem is that now it interprets `users/sign_in` as if `sign_in` is the id of a user.

My original simple idea is that each user would only be able to see his or her own to-do items. In that case, there is no need to nest the items under user, since there would only be a single list of items relevant at any time (those of the current_user).

Comment (brittanyjspringer) [reply config/routes.rb]: @DarylMcCullough per my Bloc message back to you, I'm fine with keeping it the way it is. Looking forward to your next commit :)

**Comment by brittanyjspringer** [comment]
@DarylMcCullough looks like you just have the item_spec.rb to handle. I'll merge through when you re-assign it to me. Do note that I will be away from my computer in the morning tomorrow (Sunday).

**Comment by brittanyjspringer** [comment]
Item Spec tests are on checkpoint-6. Please be careful about which branch you are/pushing to. Merging in.

**Review by brittanyjspringer — APPROVED** [review]

**Pull request merged**
